app/controllers/pdl.py

```diff
@@ -341,6 +341,22 @@ def titles_title_i_subtitles_subtitle_e_summary_search():
     endpoint_response = generate_title_i_summary_response(subtitle_id, start_year, end_year)
     return endpoint_response
 
+# /pdl/titles/title-ii/summary:
+def titles_title_ii_summary_search():
+    start_year = 2014
```

Comment:
For now, please use 2018 to 2022 as the start and end years for consistency. We are using that for individual programs under Title II.